apps/mobile/app/src/main/java/com/glycemicgpt/mobile/plugin/PluginRegistry.kt

```
@@ -98,12 +122,16 @@ class PluginRegistry @Inject constructor(
                 val plugin = factory.create(pluginContext)
                 plugin.initialize(pluginContext)
                 plugins[meta.id] = plugin
+                compileTimePluginIds.add(meta.id)
                 Timber.d("Plugin created: %s (%s)", meta.name, meta.id)
             } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)